TESTS/configs/mbedtls_wisun_config.h

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MBEDTLS_WISUN_CONFIG_H_
+#define MBEDTLS_WISUN_CONFIG_H_
+
+#define MBEDTLS_ENTROPY_MAX_SOURCES 20       // save 320 bytes of RAM (default is 20 entropy source)
+#define MBEDTLS_SSL_MAX_CONTENT_LEN 8192    // Use 4kB buffers for input and output instead of 16kB
```

Comment:
The comment says "4kB", but the define is set to 8kB - which one is right?

Reply:
Done